src/coreneuron/sim/scopmath/crout_thread.hpp

```diff
@@ -132,56 +132,29 @@ inline void nrn_scopmath_solve_thread(int n,
                                       int* y,
```

Comment:
```suggestion
                                      int* /* y */,
```
given the large number of macros, it's very hard to know if `y` is really used or not.